__tests__/newsApp.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../newsApp");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  // console.log('test seeding done');
```

Comment:
Don't forget to remove any commented out code or console.logs before creating your pull request